src/System.Runtime.CompilerServices.Unsafe/ref/System.Runtime.CompilerServices.Unsafe.cs

```diff
@@ -39,6 +39,7 @@ public static partial class Unsafe
         public static T ReadUnaligned<T>(ref byte source) { throw null; }
         public static unsafe T ReadUnaligned<T>(void* source) { throw null; }
         public static unsafe T Read<T>(void* source) { throw null; }
+        public static void SkipInit<T>(out T value) where T : struct { throw null; }
```

Comment:
This should not be constrained as unmanaged.

Reply:
Nothing else is on Unsafe is constrained that way, and the constrain just makes the method less useful for no good reason.

Reply:
It's already not constrained as `unmanaged`, I've given the feedback to API review and will keep this as `where T : struct` for now.

Reply:
Constraining it as `struct` is still unnecessary. The problem with unnecessary nice-to-have constains methods is that it makes the method impossible to use in other generic code that does not have `T` constrained.

Reply:
We do have a lot of APIs on `Unsafe` constrained to `where T : struct` however. While this wouldn't "fail" for a `ref` (I believe it would just keep whatever value was already there, either `null` or some valid handle), it still seems undesirable.

> The method impossible to use in other generic code that does not have T constrained.

Perhaps this is something we should look at addressing instead? Some way for users to say `if (T.IsValueType) { /* Can now call generic methods constrained to struct */ }` ?

Reply:
I'll give the feedback to API review when we are done with the current issue.

Reply:
API review agreed that it was fine to remove the constraint. I've updated the PR.

Will work on adding tests after lunch.